mdsuite/calculators/green_kubo_ionic_conductivity.py

```diff
@@ -169,6 +175,7 @@ def _calculate_prefactor(self):
         -------
 
         """
+        # TODO improve docstring
```

Comment:
TODO improve docstring

Reply:
This will also be changed in the new API for which there is a current PR waiting on this PR